packages/@okta/vuepress-site/docs/guides/add-an-external-idp/add-okta-redirect-uri-to-idp/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Add Okta redirect URI to the Identity Provider
+---
+The redirect URI sent in the authorize request from the client needs to match the redirect URI in the Identity Provider. This is the URL where the Identity Provider returns the authentication response (the access token and the ID token). It needs to be a secure domain that you own.
+
+<StackSelector snippet="redirecttoidp" />
```

Comment:
Looks like this is missing a snippet for SAML, so appears broken when you have selected SAML on previous steps.

If SAML doesn't have to do anything here, we should probably explain that.